src/ZODB/DemoStorage.py

```diff
@@ -308,10 +308,9 @@ def store(self, oid, serial, data, version, transaction):
 
         if old != serial:
             rdata = self.tryToResolveConflict(oid, old, serial, data)
-            self.changes.store(oid, old, rdata, '', transaction)
-            return ResolvedSerial
-
-        return self.changes.store(oid, serial, data, '', transaction)
+            self.changes.store(oid, old, rdata, '', transaction, True)
```

Comment:
``self.changes`` could be anything. You can't assume it takes the resolved argument.

Why not let ``DemoStorage`` have its own ``_resolved``?